.gitignore

```diff
@@ -1,5 +1,7 @@
 .idea/*
 bower_components
+node_modules/
```

Comment:
ignore installed npm modules. currently we have local bower here but it may be more modules later on